runtime/syntax/nftables.yaml

```diff
@@ -0,0 +1,19 @@
+filetype: nftables
+
+detect:
+    filename: "nftables.conf$"
+    header: "^#.*?/etc/nftables"
+    
+rules:
+    - statement: "(chain|define|table)"
+    - symbol: "(=)"
+    - diff-modified: "(flush|type|counter)"
+    - special: "^\\[(Unit|Install|Service|Socket|Timer)\\]"
```

Comment:
I think this line was copied in another file but it is not in nftables syntax, so it would have to be removed.
```suggestion
```

Reply:
Oh yeah i've copied another file and changed something. I will change it, when i will be at home.